src/PassengerClass.js

```diff
@@ -0,0 +1,16 @@
+const { bookshelf, Passenger, Passengers, knex } = require('../database/connection.js')
+const commands = require('./commands.js')
+
+class PassengerClass {
```

Comment:
Is it not explicit based on where it's imported from?  Are you using both `Passenger` and `PassengerClass` in the same file?  If so, I suspect there's a different type of code smell happening, but you could mitigate by using namespaces:

models 
|-- Passenger.js
|-- index.js

index.js in models would just export all of the Model objects as an object, then you could:
```js
const Models = require( './models' )
Models.Passenger.whaevs();
```